app/src/main/java/com/framgia/englishconversation/screen/imagedetail/ImageDetailViewModel.java

```diff
@@ -64,6 +64,11 @@ public boolean onItemLongClick(View viewGroup, TimelineModel item) {
                     return false;
                 }
 
+                @Override
```

Comment:
ERROR: 'method def modifier' have incorrect indentation level 16, expected level should be one of the following: 8, 12.